### PR TITLE
feat: update keywords

### LINKED ISSUE #37: List of new keywords
Meta issue, where we can track new keywords to be added

- [x] https://github.com/flix/highlightjs-flix/issues/36
- [x] https://github.com/flix/highlightjs-flix/issues/35
- [x] https://github.com/flix/highlightjs-flix/issues/34
- [x] https://github.com/flix/highlightjs-flix/issues/33

### PR DESCRIPTION
**Pull request by jaschdoc**
Closes https://github.com/flix/highlightjs-flix/issues/37 and sub-issues

### CODE REVIEW

**Comment by jaschdoc** [comment]
Unverified commit is OK, I made it in a github codespace that doesn't have my gpg key

File: src/flix.js [hunk 1]
```diff
@@ -77,66 +77,59 @@ export default function(hljs) {
...
-      "type",
       "alias",
-      "yield",
-      "lazy",
-      "force",
-      "override",
+      "as",
+      "case",
+      "catch",
+      "deafult",
```
Comment (magnus-madsen) [inline src/flix.js:84]: oops :)

Comment (magnus-madsen) [reply src/flix.js:84]: (I forget where the default keyword is used).

Comment (magnus-madsen) [reply src/flix.js:84]: (Its probably fine-- if you decide to do another PR then I would sync with the content of `flix.tmLanguage.json`  from `flix/vscode-flix`.

**Pull request merged**